### DISCUSSION TITLE
Download EBAS data directly

### DISCUSSION THREAD

**Discussion by dominichr1** [pangeo-data/escience-2022#11] [Q&A] [answered]
Previously, it was possible to download EBAS data via OPeNDAP (Open-source Project for a Network Data Access Protocol) for surface in situ (OPeNDAP service) http://dev-ebas-pydap.nilu.no/ using the following:

`ds = open_dods(
'http://dev-ebas-pydap.nilu.no/' 
'NO0042G.dmps.IMG.aerosol.particle_number_size_distribution'
'.1h.NO01L_NILU_DMPSmodel2_ZEP.NO01L_dmps_DMPS_ZEP01.2.dods')`

see previous schools (https://nordicesmhub.github.io/NEGI-Abisko-2019/training/example_notebooks/from_ebas_using_pydap.html)

I get the following error now:

HTTPError: 404 Not Found
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /NO0042G.dmps.IMG.aerosol.particle_number_size_distribution.1h.NO01L_NILU_DMPSmodel2_ZEP.NO01L_dmps_DMPS_ZEP01.2.dods was not found on this server.</p>
<hr>
<address>Apache/2.4.7 (Ubuntu) Server at dev-ebas-pydap.nilu.no Port 80</address>
</body></html>

**Comment by MichaelSchulzMETNO** [comment, marked as answer]
I put an example that uses the new EBAS interface here
https://github.com/pangeo-data/escience-2022/blob/main/tutorial/examples/notebooks/from_ebas_using_xarray.ipynb

**Reply by dominichr1** [reply]
thanks, sorry I missed this great
